v1/tasks/errors.go

```diff
@@ -5,29 +5,8 @@ import (
 	"time"
 )
 
-// ErrRetryTaskLater ...
-type ErrRetryTaskLater struct {
-	name, msg string
-	retryIn   time.Duration
-}
-
-// RetryIn returns time.Duration from now when task should be retried
-func (e ErrRetryTaskLater) RetryIn() time.Duration {
-	return e.retryIn
-}
-
-// Error implements the error interface
-func (e ErrRetryTaskLater) Error() string {
-	return fmt.Sprintf("Task error: %s Will retry in: %s", e.msg, e.retryIn)
-}
-
-// NewErrRetryTaskLater returns new ErrRetryTaskLater instance
-func NewErrRetryTaskLater(msg string, retryIn time.Duration) ErrRetryTaskLater {
-	return ErrRetryTaskLater{msg: msg, retryIn: retryIn}
-}
-
-// Retriable is interface that retriable errors should implement
```

Comment:
thanks for fixing this cursed word most of all

Reply:
I thought that I'd invented this spelling and was sitting here wondering what was wrong with me, but I just checked and it actually wasn't me and I feel a whole lot better about myself now.